### ISSUE TITLE
Please consider to put 'while' at the top

### ISSUE THREAD

**Issue by volkan-yazilim** [dotnet/docs#9857]
Both in this page and in the menu.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7a02f9ec-957b-e05f-36b7-f4ae5521606e
* Version Independent ID: 6cf01121-3d2d-8825-bd74-6f785a0da7d6
* Content: [Iteration Statements - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/iteration-statements)
* Content Source: [docs/csharp/language-reference/keywords/iteration-statements.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/iteration-statements.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by volkan-yazilim** [comment]
It is not nice neither polite to delete comments.

**Comment by BillWagner** [comment]
@volkan-yazilim I agree, and we don't delete comments. We closed this issue because we don't plan to address it. It's still visible. If you and others feel strongly, you can advocate for the change, and we'll reconsider.
